### PR TITLE
CMS: templates: add guide links and fixes other link in about cms

### PR DESCRIPTION
**Pull request by katilp**
Addresses #1787 and #1937

### CODE REVIEW

**Comment by katilp** [comment]
Adding the "Guide" links is important, but as they are there now with this PR, it reads very strange (the page starts with introduction to CMS, then deviates to the guide links and continues again with the core subject of the page: the open data...). It would be best to have some graphical buttons for the guides so that they would be more "separate" from the normal text.

File: cernopendata/modules/fixtures/data/articles/about-cms/cms.md [hunk 1]
```diff
@@ -36,7 +42,8 @@ The following are provided through this portal:
     * Definition of same objects is different in each analysis.
 
 * The files can be read in [ROOT](http://root.cern.ch/), but they cannot be opened (and understood) as simple data tables.
-* Only the runs that are validated by data quality monitoring should be used in any analysis. The [list of the validated runs](/collection/CMS-Validation-Utilities) is provided.
+* Only the runs that are validated by data quality monitoring should be used in any analysis. The [list of the validated runs](/search?page=1&size=20&q="CMS list of validated runs") (FIXME markdown does not accpet hyphen in links?) is provided.
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/about-cms/cms.md]: Try this: `/search?page=1&size=20&q="CMS%20list%20of%20validated%20runs"`

File: cernopendata/modules/fixtures/data/articles/about-cms/cms.md [hunk 2]
```diff
@@ -36,7 +42,8 @@ The following are provided through this portal:
...
+* A small sample of [RAW data](/search?page=1&size=20&q=&experiment=CMS&file_type=raw) is also provided.
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/about-cms/cms.md]: Should this be "raw" rather than "RAW"?

Comment (katilp) [reply cernopendata/modules/fixtures/data/articles/about-cms/cms.md]: We are sort of used to RAW in CMS (i.e. RAW, RECO, AOD), but indeed here probably "raw data" is better

**Comment by katilp** [comment]
Thanks, updates done!

**Comment by katilp** [comment]
For merging, let me know if I should so this differently 😃

**Comment by RaoOfPhysics** [comment]
@katilp: In your original post I think one of the issue numbers is wrong. And maybe you can remove the bit about "FIXME"? Thanks!

**Comment by ioannistsanaktsidis** [comment]
Closing as superseded by #1941

**Pull request closed without merging**